docs/source/README.md

```diff
@@ -86,7 +86,13 @@ conda config --add channels conda-forge
 
 Next, `HATCHet` can be installed with the following one-time command:
 ```shell
-conda install hatchet
+conda install hatchet tabix mosdepth
```

Comment:
The end user should just have to do `conda install hatchet` and we can specify `tabix/mosdepth` etc. in the `meta.yml` for the bioconda recipe.

Reply:
Tabix and mosdepth added to conda recipe for HATCHet 1.0.0